service/controller/app/v1/key/error.go

```diff
@@ -11,6 +11,15 @@ func IsEmptyValueError(err error) bool {
 	return microerror.Cause(err) == emptyValueError
 }
 
+var executionFailedError = &microerror.Error{
+	Kind: "executionFailed",
+}
+
+// IsExecutionFailed asserts executionFailedError.
+func IsExecutionFailed(err error) bool {
```

Comment:
Please check `aws-operator` as reference implementation and see https://github.com/giantswarm/fmt/blob/master/go/errors.md. We do not want to have the matcher for this and the error should move to the top with a comment.

Reply:
Thanks, I removed the matcher function and also change the unittest to not to match with this specific error.